cmd/cork/create.go

```diff
@@ -52,6 +53,8 @@ func init() {
 		"chroot", "chroot", "SDK chroot directory name")
 	createCmd.Flags().StringVar(&manifestURL,
 		"manifest-url", coreosManifestURL, "Manifest git repo location")
+	createCmd.Flags().StringVar(&manifestBranch,
+		"manifest-branch", "master", "Manifest git repo branch")
```

Comment:
While we are at it might as well pass through `--manifest-name=` as well, making this work for CoreOS release branches which need `-b build-123 -m release.xml` to set things up correctly.